packages/fluentui/react-northstar/src/themes/teams/components/Pill/pillStyles.ts

```diff
@@ -0,0 +1,39 @@
+import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
+import { PillStylesProps } from '../../../../components/Pill/Pill';
+import { PillVariables } from './pillVariables';
+import { getBorderFocusStyles } from '../../getBorderFocusStyles';
+
+export const pillStyles: ComponentSlotStylesPrepared<PillStylesProps, PillVariables> = {
+  root: ({ props: p, variables: v, theme: { siteVariables } }): ICSSInJSStyle => {
+    return {
+      display: 'inline-flex',
+      borderRadius: v.borderRadiusCircular,
+      background: v.background,
+      margin: `${v.horizontalMargin} ${v.verticalMargin}`,
```

Comment:
```suggestion
      margin: v.margin,
```

Let's just use `v.margin` is there is no other requirements